### ISSUE TITLE
include ros services and actions

### ISSUE THREAD

**Issue by TKaluza** [RobotecAI/ros2-for-unity#48]
Would be great to have also [ros2 services ](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Services/Understanding-ROS2-Services.html) or [ros2 actions](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Actions/Understanding-ROS2-Actions.html) available! 
Thank you for your create unity asset!

**Comment by adamdbrw** [comment]
@TKaluza I agree these would be great features! We have an issue for services here (in associated repository): https://github.com/RobotecAI/ros2cs/issues/1 - it would be a valuable PR! 
Supporting services is something we might still do this year, but since we are not using them ourselves, it would be great to have a contribution.

**Comment by adamdbrw** [comment]
Services are now supported, I will rename the issue to reflect that

**Comment by samiamlabs** [comment]
Is anyone currently working on adding support for actions?
I have some time over and can give it a shot if is is not already in progress :)

**Comment by adamdbrw** [comment]
@samiamlabs No work that I know of is being done on actions at the moment, would be great if you did it!
